src/assets/tailwind.css

```diff
@@ -9,7 +9,10 @@
 .active {
   @apply bg-gradient-to-br from-purple-500 to-indigo-500 text-white;
 }
+.card-button {
+  @apply block rounded-t-none mt-4 w-full bg-gradient-to-br from-purple-500 to-indigo-500 text-purple-200   font-semibold font-heading text-lg text-center px-5 py-5;
+}
```

Comment:
don't add classes like this.
Directly add this to the react component

Reply:
okay.is there any reason for this ?